src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -43,6 +46,7 @@ subroutine CStateUpdateDynPatch(bounds, num_soilc_with_inactive, filter_soilc_wi
     !
     ! !DESCRIPTION:
     ! Update carbon states based on fluxes from dyn_cnbal_patch
+    ! This routine is not called with FATES active.
```

Comment:
the call to execute this routine is encapsulated in a use_fates logical already, calling use_fates inside the routine was redundant. I added the text here so that the user is aware of this, and does not think its doing calculations that are fates relevant.

Reply:
Ahhh, this is only called under DynamicAreaConservation, and that's only called for BGC, and not FATES. And what you did was to remove the extra check on FATES, since it's not needed. Now, that I get it, this is clear and a nice change.